tests/testCases/negativeTC.spec.js

```diff
@@ -0,0 +1,17 @@
+const { test, expect } = require('@playwright/test');
+import { mainPage } from '../demoQA/pages/pageObject';
+import { filler } from '../demoQA/pages/inputs';
+
+test('Negative scenario', async({page}) => {
```

Comment:
Let's give more understandable names to tests, for example `Negative scenario: Enter incorrect email`